operations/mimir-mixin/docs/playbooks.md

```diff
@@ -1130,6 +1130,158 @@ The limit protects the system’s stability from potential abuse or mistakes, an
 
 > **Note**: Invalid metrics metadata are skipped during the ingestion, and valid metadata within the same request are ingested.
 
+### err-mimir-ingester-max-ingestion-rate
+
+This critical error occurs when the rate of received samples per second is exceeded in an ingester.
+
+The ingester implements a rate limit on the samples per second that can be ingested, and it's used to protect an ingester from overloading in case of high traffic.
+The limit is a per-instance limit and it's applied on all samples received, across all tenants, in each ingester.
+
+How to **fix**:
```

Comment:
```suggestion
How to **fix** it:
```
Please add a placeholder object to any phrase that is transitive (one that requires an object).

Reply:
I think we use `How to **fix**:` across plenty of places. I can submit a different PR that changes it to `How to **fix** it:`

Reply:
I just pushed a commit to replace it across the whole runbooks file.